<launch>

	<arg name="simulation"   default="true"/>
	<arg name="display" default="true"/>
	<arg name="cam_name" default="camera_right_eye"/>
	<arg name="cam_number" default="0"/>

	<!-- - - - - - - - - - - - N a v i g a t i o n - - - - - - - - - - -->
	<include file="$(find bender_nav)/launch/complete_nav.launch">
		<arg name="simulation" value="$(arg simulation)"/>
		<arg name="rviz" value="$(arg display)"/>
	</include>


	<!-- Hardware -->
	<include file="$(find bender_sensors)/launch/camera/$(arg cam_name).launch">
		<arg name="display" value="false"/>
		<arg name="cam_number" value="0"/>
	</include>

	<!-- Face Detector -->
	<include file="$(find bender_vision)/launch/face/face_detector.launch">
		<arg name="cam_name" value="$(arg cam_name)"/>
		<arg name="display" value="$(arg display)"/>
	</include>

</launch>